Dockerfile

```diff
@@ -191,6 +191,12 @@ COPY --chmod=755 configs/rcon.yaml /home/steam/steamcmd/rcon.yaml
 COPY --chmod=755 configs/PalWorldSettings.ini.template /
 COPY --chmod=755 gosu-amd64 /usr/local/bin/gosu
 
+RUN chown -R "$PUID:$PGID" "/home/steam/" && \
```

Comment:
I understand you are new to docker and might not have run into theese differences, so im gonna explain it for you:

This is bad and can not work, at this time the files are owned by a user and an PGID/PUID that is different from the user we later use when we use docker-run and the entrypoint.sh. There is a massive difference between what happens in docker build (image creation) and docker run (run instance as container from image). 

The idea of the user-jail mechanic with the ability to hot-swap the IDs over Docker-Compose is that it doesnt matter which one you use, because the files will be setup in permissions at "start-time", because CMD/ENTRYPOINT wont be run at docker-image build time, only at docker run time.

Please read theese articles:
https://medium.com/@mccode/understanding-how-uid-and-gid-work-in-docker-containers-c37a01d01cf
https://spacelift.io/blog/docker-entrypoint-vs-cmd

In essence you might just want explain what you want to do, rather then do it in a kinda bad way, i dont want to insult you or your knowledge, sorry if it feels like that, but user-jail mechanic via IDs is a rather very advanced topic in writing docker images the right way.

Reply:
Just want to say I'm not ignoring this. I really appreciate the resources to read. Just haven't had the time to read and play around in docker to ensure I fully understand.